File: 2019_1/leonardo/API/models/service.model.js
module.exports = (sequelize, DataTypes) => {
	var Model = sequelize.define('Service', {
		type: DataTypes.STRING,
		description: DataTypes.STRING
	});

	Model.prototype.toWeb = function(pw) {
		let json = this.toJSON();
		return json;
	};

	return Model;
};

File: 2019_1/leonardo/Aplicativo/lib/Models/Service.model.dart
class Service {
  int id;
  String type;
  String description;

  Service({this.id, this.type, this.description});

  Service.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    type = json['type'];
    description = json['description'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['type'] = this.type;
    data['description'] = this.description;
    return data;
  }
}